docs/how-to-guides/deploy-feathr-api-as-webapp.md

```diff
@@ -0,0 +1,108 @@
+# Feathr API
+The API currently supports following functionality
+
+1. Get Feature by Qualified Name
+2. Get Feature by GUID
```

Comment:
Maybe we should make it clear that the result is directly from backend system (purview)

Reply:
We will make sure to address in Yihui's PR